src/prompts/underlay.scss

```diff
@@ -0,0 +1,19 @@
+body.kumulos-underlay-blur {
+    overflow: hidden;
+}
+
+.kumulos-underlay-blur > *:not(.kumulos-prompt):not(.kumulos-overlay) {
```

Comment:
`:not(.kumulos-overlay)`, is that just copied and missed or should it target that?

Wonder if we can/should refactor the permission Overlay component itself (maybe rename PermissionOverlay for clarity) to share this? Or rather use existing CSS and refactor the underlay to use same.

Reply:
Have just repurposed the `<Underlay />` to now a `<BackgroundMask />` which can render children.  This can be used alone (as in the existing case of the prompt underlay config) and is now used as the parent element that was in effect within the `<Overlay />` component too

Reply:
Nice one!